proposal_stages.md

```diff
@@ -0,0 +1,28 @@
+# ROS 2 Production Proposal Stages
+The following stages are also intended to be used as github tags for tracking.
+## Stage 1: Draft
+
+The draft stage represents the initial proposal of a project or task which has been formalised into a github issue for presentation to the community. To move to the next stage, this draft should be presented to the ROS2 Production WG and discussed in relevant channels (e.g slack, github, discourse, etc.), having addressed feedback about the scope, methodology, and tasks required to implement this proposal. 
+
+## Stage 2: Ready
+
+In this stage, the draft has received feedback from the community. It is considered to be in “ready for work” state. Tasks have been clearly defined and broken down into relevant sub-components and subtasks, sufficient such that a principal contributor may begin execution with clear direction. To move to the next stage, there should be a discussion/vote on the proposal’s priority, who will be the principal contributor, and the intended timeline for completion. 
+
+
+## Stage 3: Accepted
+
+In this stage, the proposal has been accepted and is part of the ROS 2 Production WG’s project planning board. A principal contributor has been identified. Accepted proposals will be visible on the working group’s project roadmap. To move to the next stage, the principal contributor should be providing regular updates to the working group. 
+
+## Stage 4: In Progress
```

Comment:
Feedbacks from March 6th WG meeting.

At a certain point the proposal should also be discussed with the ROS 2 community.
The type of interaction should depend on the impact and type of proposal.

For example, for a proposal that modifies a single ROS 2 package, it may be sufficient to consult with its maintainers.
More complex proposals may require escalation to the broader community (discourse) and/or the ROS 2 TSC.

Getting feedbacks from maintainers shouldn't be a blocker to start development, but it should be done as soon as possible, for example inviting them to review meetings in the WG.